guides/writing-addons/dependencies.md

```diff
@@ -1 +1,407 @@
-<!-- Should cover where to put dependencies in package.json, how to export specific assets -->
+### NPM/Yarn and Bower Configuration
+
+Ember CLI supports [NPM](https://www.npmjs.com), [yarn](https://yarnpkg.com/)
+and optionally [Bower](http://bower.io/) for dependency management. It will
+detect whether you are using npm or yarn by the presence of a `yarn.lock` file
+in your project.
+
+A newly generated Ember CLI project only has NPM dependencies, so you will
+notice a `package.json` file at the root of your project, but not a `bower.json`.
+To use Bower packages, you will have to first run `bower init` to create a
+`bower.json` file also at the root of the project.
```

Comment:
It should be not recommended to use bower. It is not officially deprecated, but it should be.